gulp/config.js

```diff
@@ -1,11 +1,66 @@
-'use strict';
+var dest = "./dist/assets/";
```

Comment:
There's a good mixture of both ES5 and ES6 in this PR. 

For Node.js code, we can freely use ES6, so things like `var` can almost never be used (favoring `let` and `const` instead), and we can also use things like arrow functions to make code easier to read. I'd suggest we pick one version of thelanguage and stick to it. I'd favor ES6 for all node.js related code as it's easier to read, but we can use whatever people feel most comfortable with.

We can open a new issue to add ESLint to the repo, which will help enforce a standard.

Reply:
P.S - we can always fix this up in a followup PR 👍

Reply:
Yeah, there was definitely some copying and pasting going on from previous projects I've worked on without paying much attention to ES6 syntax.